File: Food App/app/user-info.ts
export interface UserInfo {
    ID?:string,
    name?:string,
    email?:string,
    password?:string
}
export interface UserInfoNew {
    id?:string,
    name?:string,
    role?: string,
    location?:string,
    email?:string,
    password?:string
}

File: 13-05-25/TaskManager/UserInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TaskManager
{
    internal class UserInfo:Project
    {
        public string UserName { get; set; }
        public string Password { get; set; }
        public string Email { get; set; }
        public string Role { get; set; }
        public UserInfo(string title, string description, UserInfo manager, string userName, string email, string password, string role)
        : base(title, description, manager)
        {
            UserName = userName;
            Email = email;
            Password = password;
            Role = role;
        }
        public override string ShowDetails()
        {
            return string.Format($"{UserName}-{Email}--{Role}");
        }


    }
}

